src/Atoms/Inputs/Input.js

```diff
@@ -134,13 +136,15 @@ const Input = ({ className, input, type, ...rest }: PropsType) => {
           }${rest.error || ''}`}
         </InputLabel>
         <InputWrapper {...input} {...rest} type={type} />
+        {rest.renderSuffix && rest.renderSuffix()}
```

Comment:
I think we want to take `renderSuffix` from spreading the `prop` rather than `rest` that way it does not get passed further down.